.gitignore

```diff
@@ -24,3 +24,4 @@
 
 /tests/runtime
 !/tests/runtime/.gitkeep
+/.idea/
```

Comment:
Directory `.idea` should be in your homedir `.gitignore` config (for all your projects).